### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by atenaiis**
testing linterns

### CODE REVIEW

**Comment by atenaiis** [comment]
Team,

This is my Weather app, please follow the readme file instructions .

cheers!!

**Review by umararshadbut — COMMENTED** [review]
# Status: INVALID :red_circle: 

**Hi @atenaiis **
The readme is descriptive :clap: your commit messages are perfect :clap: However there is a problem with your git-flow and pull request description. Please do not merge the pull request before approving the project. And add the description in the pull request.
I am going to mark this project as invalid so when you fix the issue just submit another code review request.
##

 **If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**Umair**

**Comment by atenaiis** [comment]
Team
 this is a weather app to display current information from a specific place. Using JS connected with a third-part API and applying concepts about Promises, Async/Await syntax, and API connection.
Please follow Readme file instructions.

Cheers!

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Atenais Campos García

THIS IS VERY IMPRESSIVE WORK. YOU HAVE DONE A GOOD JOB :+1:

## To Highlight :tada:


1. Your readme is descriptive.
2. You have descriptive git commit messages.


Good job on implementing the changes requested. However, there are some changes that you need to make in order to get your project approved

- [x] Make the develop branch default branch

- [x] remove commented code

- [x] There should be a toggle control to determine if the result will be shown in Celsius or Fahrenheit  [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png)

- [x] Kindly add  ```"start": "webpack-dev-server --mode development --open" // optional``` to your package.json scripts, or remove ```npm run start``` from your readme instructions

- [x] There should be error handling for promises or async/await [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png)



## OPTIONAL

- [ ] It would be nice if you add a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API 

- [ ] it would be nice if it fetches the current geolocation of the user when the app is first accessed and fetches the weather from that location

Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+> Follow these steps to set up and run the quickstart:
+  - Clone/Download this repo and open this folder in a Terminal.
+    `git clone git@github.com:atenaiis/Weather-app.git`
+  - Install the Nodejs:
+  [https://nodejs.org/en/download/](https://nodejs.org/en/download/)
+
+  - Run npm install on your terminal *be sure that you are in the project's directory*:
+  `npm intall`
+  - Finally run a local server using chrome:
+  `npm run start`
```
Comment (MkrtichSargsyan) [inline README.md]: There is no ```start``` script in the package.json file

File: src/app.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <div class="my-3">${weather.WeatherText}</div>
+    <div class="display-4 my-4">
+      <span>${weather.Temperature.Metric.Value}</span>
+      <span>&deg;C</span>
+    </div>
+  `;
+
+  // update the night/day & icon images
+  // const iconSrc = `../img/icons/${weather.WeatherIcon}.svg`;
+  // icon.setAttribute('src', iconSrc);
```
Comment (MkrtichSargsyan) [inline src/app.js]: Kindly remove the commented code

File: src/forecast.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+  async updateCity(city) {
+    const cityDets = await this.getCity(city);
+    const weather = await this.getWeather(cityDets.Key);
+    return { cityDets, weather };
+  }
+
+  async getWeather(id) {
+    const query = `${id}?apikey=${this.key}`;
+    const response = await fetch(this.weatherURI + query);
```
Comment (MkrtichSargsyan) [inline src/forecast.js:16]: Kindly add a try-catch block

File: src/forecast.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    const data = await response.json();
+    return data[0];
+  }
+
+  async getCity(city) {
+    const query = `?apikey=${this.key}&q=${city}`;
+    const response = await fetch(this.cityURI + query);
```
Comment (MkrtichSargsyan) [inline src/forecast.js:23]: Kindly add a try-catch block

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: CHANGES REQUESTED🛑🛑🛑🛑**

Hello @atenaiis 
Great work making the requested changes👏👏👏👏

You can merge👏👏

Happy coding 💻💻

**Pull request merged**